docs/usage/cvc_guide.rst

```diff
@@ -233,6 +233,12 @@ you can increase this number by using the ``--pool-size`` option (``-p`` for sho
 
     $ cvc download --pool-size 50  # download 50 frames at a time
 
+When using a global cache, it may be necessary to run the download command using the
```

Comment:
Please describe the circumstances under which this is necessary (i.e. if the cache is on a different volume)